### PR TITLE
changes made for case classes with trait

### PR DESCRIPTION
**Pull request by chidamCJ**

### CODE REVIEW

**Comment by chidamCJ** [comment]
added for first five graphs

File: mgwrapper/src/main/scala/mg/wrapper/charts/MGChartAttribs.scala [hunk 1]
```diff
@@ -1,5 +1,77 @@
...
+}
+case class SmallRangeOfIntegers(interpolate: Any,
+                                baseAttributes: MGChartAttribs)
+    extends BaseChartAttributes {
+  override def getAttributes: Array[(String, Any)] =
+    Array(
+      "interpolate" -> interpolate,
+    ).++(baseAttributes.getAttributes)
+}
+case class LinkedGraphic(baseAttributes: MGChartAttribs)
```
Comment (prassee) [inline mgwrapper/src/main/scala/mg/wrapper/charts/MGChartAttribs.scala]: if a case class does not take any other arguments except baseAttributes no need to create a case class for the same

File: mgwrapper/src/main/scala/mg/wrapper/charts/MGChartAttribs.scala [hunk 2]
```diff
@@ -1,5 +1,77 @@
...
+    extends BaseChartAttributes {
+  override def getAttributes: Array[(String, Any)] = {
+    Array(
+      ).++(baseAttributes.getAttributes)
+  }
+}
+case class OtherLinkedGraphic(baseAttributes: MGChartAttribs)
```
Comment (prassee) [inline mgwrapper/src/main/scala/mg/wrapper/charts/MGChartAttribs.scala]: Same as previous comment

File: mgwrapper/src/main/scala/mg/wrapper/charts/MGChartAttribs.scala [hunk 3]
```diff
@@ -1,5 +1,77 @@
...
+  override def getAttributes: Array[(String, Any)] =
+    Array(
+      "format" -> format,
+      "target" -> target,
+      "show_secondary_x_label" -> show_secondary_x_label,
+      "show_confidence_band" -> show_confidence_band,
+      "x_extended_ticks" -> x_extended_ticks
+    ).++(baseAttributes.getAttributes)
+}
+case class SmallRangeOfIntegers(interpolate: Any,
```
Comment (prassee) [inline mgwrapper/src/main/scala/mg/wrapper/charts/MGChartAttribs.scala]: please find appropriate data type instead of Any

**Comment by chidamCJ** [comment]
four case classes for seven charts

**Pull request merged**